Test/Tests/RealtimeClientChannelTests.swift

```diff
@@ -3340,8 +3340,38 @@ class RealtimeClientChannelTests: XCTestCase {
         expect(channel.state).toEventually(equal(ARTRealtimeChannelState.attached), timeout: testTimeout)
     }
 
-    // RTL7c
-    func test__110__Channel__subscribe__should_result_in_an_error_if_channel_is_in_the_FAILED_state() throws {
+    // RTL7h
+    func test__109b__Channel__subscribe__should_not_implicitly_attach_the_channel_if_options_attachOnSubscribe_is_false() throws {
+        let test = Test()
+        let client = ARTRealtime(options: try AblyTests.commonAppSetup(for: test))
+        defer { client.dispose(); client.close() }
+        
+        let channelOptions = ARTRealtimeChannelOptions()
+        channelOptions.attachOnSubscribe = false
+        let channel = client.channels.get(test.uniqueChannelName(), options: channelOptions)
+
+        let exception = tryInObjC {
```

Comment:
This doesn't really seem related to the rest of the test. If you want to have it there I’d suggest putting a comment for it. But really it seems like it's testing a different piece of functionality.

Reply:
https://github.com/ably/ably-cocoa/pull/1976/commits/1625e1b4567972f42d9b5887ff30cb9802ddc572

Reply:
One of these checks is still there

Reply:
It's now in a separate test `test__140__ChannelOptions__options_provided_when_instantiating_a_channel_should_be_frozen` (with the rest of those similar checks).

Reply:
Ah, missed that 👍